R-package/NAMESPACE

```diff
@@ -22,6 +20,8 @@ export("xgb.attributes<-")
 export("xgb.config<-")
 export("xgb.parameters<-")
 export(getinfo)
+export(print.xgb.Booster)
+export(print.xgboost)
```

Comment:
Did we remove any explicit s3 export statements?

Reply:
I have added a commit that fixes the problem by adding the roxygen tags `@method print xgboost` and `@method print xgb.Booster`